chia/server/server.py

```diff
@@ -489,7 +489,7 @@ async def api_call(full_message: Message, connection: WSChiaConnection, task_id)
                 try:
                     if self.received_message_callback is not None:
                         await self.received_message_callback(connection)
-                    connection.log.info(
+                    connection.log.debug(
```

Comment:
this removed the only way we can confirm our remote harvesters are working unless switching to DEBUG log level.  from the frying pan into the fire...